src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -144,17 +146,20 @@ def execute(self, reqStatus):
 
         # Call _execute() and feed the relevant pipeline with the objects popped from requestRecords
         try:
-            totalNumRequests, cleanNumRequests, archivedNumRequests = self._execute(requestRecords)
+            totalNumRequests, cleanNumRequests, normalArchivedNumRequests, forceArchivedNumRequests = self._execute(requestRecords)
             msg = "\nNumber of processed workflows: %s."
             msg += "\nNumber of properly cleaned workflows: %s."
-            msg += "\nNumber of archived workflows: %s."
+            msg += "\nNumber of properly archived workflows: %s."
```

Comment:
Maybe here too? Something like "Number of normally archived..."

Reply:
Ok